torch/_dynamo/variables/builtin.py

```diff
@@ -1172,6 +1172,25 @@ def call_id(self, tx, *args):
         else:
             unimplemented(f"call_id with args {args}")
 
+    def _raw_value_to_format(self, value):
+        if value.is_python_constant():
+            return value.as_python_constant()
+        elif isinstance(value, variables.TensorVariable):
```

Comment:
It prints FakeTensor stuff, I don't know how much value it is for end users?

Reply:
Not sure about how much value it is for end users, but what do you propose to do here instead? Something like <tensor> without details? 

Note that the motivating use case does have a tensor in the format output, so we need to support it at least to the extent that it doesn't crash.

Reply:
If the goal is to turn printing a Tensor into a no-op, then `<tensor>` might be better -- though since that is changing the semantics of the original program it should be behind a flag like `ignore_tensor_prints`.